Example/ExampleApp/MainViewController.swift

```diff
@@ -43,7 +43,10 @@ UIViewControllerTransitioningDelegate, UIPopoverPresentationControllerDelegate {
             present(vc, type: .push, animated: animated)
 
         case 1:
+            let image = #imageLiteral(resourceName: "ic_dismiss")
+            let dismissConfig = DismissButtonConfig(location: .left, style: .plain, itemType: .image(image))
             let vc = RedViewController()
+            vc.dismissConfig = dismissConfig
```

Comment:
I have chosen one case in order to present an image instead of text. Not sure though if that is the best approach. Maybe add one more row on the tableview `Modal with custom image` ?